src/theme.ts

```diff
@@ -109,6 +109,13 @@ const theme = {
       sm: '16',
       md: '24'
     }
+  },
+  identicon: {
+    diameter: {
+      sm: '16',
+      md: '32',
+      lg: '60px'
```

Comment:
why this one with px and the others without it? also, could we name it `size` instead, to be consistent with the other components?